### PR TITLE
USHIFT-1940: use different build image for arm

### PR DESCRIPTION
**Pull request by eslutsky**

### CODE REVIEW

File: test/bin/build_rpms.sh [hunk 1]
```diff
@@ -18,7 +18,7 @@ BUILD_CMDS=('make rpm')
 
 # In CI, build the current branch from source with the build tools using used by OCP
 # (only for x86_64 - no FIPS support on ARM and no build image for ARM).
-if [ -v CI_JOB_NAME ] && [ "${UNAME_M}" == "x86_64" ]; then
+if [ -v CI_JOB_NAME ]; then
```
Comment (ggiguash) [inline test/bin/build_rpms.sh:21]: Please fix the comment as well

File: Makefile [hunk 2]
```diff
@@ -112,6 +112,12 @@ GO_TEST_PACKAGES=./cmd/... ./pkg/...
 # Use an environment variable to allow CI to disable when cross-compiling.
 export CGO_ENABLED ?= 1
 
+# Specify OCP build tools image tag when building rpm with podman
+RPM_BUILDER_IMAGE_TAG := rhel-9-release-golang-1.20-openshift-4.15
+ifeq "$(strip $(shell uname -m))" "aarch64"
+	RPM_BUILDER_IMAGE_TAG := $(RPM_BUILDER_IMAGE_TAG).art-arm64
```
Comment (dhellmann) [inline Makefile:118]: Will the naming convention using this suffix always hold, or should we just use 2 full strings for clarity?

File: Makefile [hunk 3]
```diff
@@ -112,6 +112,12 @@ GO_TEST_PACKAGES=./cmd/... ./pkg/...
 # Use an environment variable to allow CI to disable when cross-compiling.
 export CGO_ENABLED ?= 1
 
+# Specify OCP build tools image tag when building rpm with podman
+RPM_BUILDER_IMAGE_TAG :	= rhel-9-golang-1.20-openshift-4.15
```
Comment (pmtk) [inline Makefile:116]: typo? `: =` -> `:=`?

File: Makefile [hunk 4]
```diff
@@ -295,17 +301,18 @@ commit: image-build-configure image-build-commit
...
+		--build-arg TAG=$(RPM_BUILDER_IMAGE_TAG) \
+		--authfile /etc/crio/openshift-pull-secret \
+		--tag microshift-builder:$(RPM_BUILDER_IMAGE_TAG) - < ./packaging/images/Containerfile.rpm-builder ; \
 	podman run \
 		--rm -ti \
 		--volume $$(pwd):/opt/microshift:z \
 		--volume $(GO_CACHE):/go/.cache:z \
 		--env TARGET_ARCH=$(TARGET_ARCH) \
-		microshift-builder:$$RPM_BUILDER_IMAGE_TAG \
+		--authfile /etc/crio/openshift-pull-secret \
```
[the remark this answers is not in the document]
Comment (eslutsky) [reply Makefile:314]: yeap probably not needed

**Pull request merged**